### PR TITLE
ui: combining andrew's fixes to axis labels

### PR DESCRIPTION
**Pull request by dianasaur323**
combining oustanding fixes from https://github.com/cockroachdb/cockroach/pull/20737

### CODE REVIEW

**Comment by dianasaur323** [comment]
nvm. i just ran make build and something is wrong. let me take a look as to why.

**Comment by dianasaur323** [comment]
Alrighty, problem solved. I made a couple edits after copying in @couchand's changes
1) You made the axisDomain.label="" for count, but this made all the labels disappear for charts that used the count label. I added this back in, with the side effect of everything having (count) but better that than no label?
2) You had a couple hanging references in there that were marked by the linter, so I removed those.

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx
Comment (vilterp) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx]: @nvanbenschoten do you think these are good labels?

**Review by vilterp — APPROVED** [review]
LGTM. Maybe will wait for @couchand to sign off though (if he has another #naptime while on vacation)

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx
Comment (vilterp) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx]: (y-axis label on time series graph)

**Comment by vilterp** [comment]
Weird. `make lint` passed for me locally.

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx
Comment (nvanbenschoten) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx]: I'm not sure I get why they're being changed, to be honest. They were more accurate before.

**Review by nvb — COMMENTED** [review]
Is the plan to merge this as is? If so, the commit messages could use a little work. Also, is this replacing https://github.com/cockroachdb/cockroach/pull/20737?

**Comment by dianasaur323** [comment]
@nvanbenschoten ha, they are being changed because I took a look at #20737 and went with those. If you like the more detailed ones, I can move them back. Essentially, I accidentally merged changes to those labels without realizing that @couchand had already made a fix in #20737 during fixit day, so now I'm trying to combine the two commits. So yes, we would go with this one, not #20737. 

For my knowledge, how could we improve the commit messages? I haven't done this before.

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx
Comment (dianasaur323) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx]: ok, swapped out the names in requests to the more detailed versions, and also fixed the linting error. will squash and change the commit message before merging. also waiting for andrew to take a look.

File: pkg/ui/src/util/format.ts
Comment (couchand) [inline pkg/ui/src/util/format.ts]: indent issue

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx
Comment (couchand) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx]: these `units` properties aren't needed, since the default prop is `AxisUnits.Count` anyway

File: pkg/ui/styl/shame.styl
Comment (couchand) [inline pkg/ui/styl/shame.styl]: is this change related?

**Review by couchand — CHANGES_REQUESTED** [review]
I think it's better to not have `(count)` appear all over the place.  It looks like the issue there was that you're missing an else if:

screen shot 2018-01-02 at 12 52 54 pm

**Comment by dianasaur323** [comment]
Removing (count) looks like this 
image

**Comment by dianasaur323** [comment]
Review status: 0 of 10 files reviewed at latest revision, 4 unresolved discussions.

---

*[pkg/ui/src/util/format.ts, line 80 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-L0f8Ikz3vqDoT_PgoiO-r3-80:-L1sd52r5W-5-N0a1sqd:b2xb9q3) ([raw file](https://github.com/cockroachdb/cockroach/blob/553e24843d68b0f543cef1085d5919f497281788/pkg/ui/src/util/format.ts#L80)):*
<details><summary><i>Previously, couchand (Andrew Couch) wrote…</i></summary><blockquote>

indent issue
</blockquote></details>

Could you clarify for my benefit? This looks similar to the below export function with a const scale declaration as well?

---

*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx, line 28 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-L0f8Ikz3vqDoT_PgoiP-r3-28:-L1sdUXA1b8Vo2ImMjTl:b-4hbcef) ([raw file](https://github.com/cockroachdb/cockroach/blob/553e24843d68b0f543cef1085d5919f497281788/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx#L28)):*
<details><summary><i>Previously, couchand (Andrew Couch) wrote…</i></summary><blockquote>

these `units` properties aren't needed, since the default prop is `AxisUnits.Count` anyway
</blockquote></details>

Done. Also got rid of this for the other dashboards.

---

*[pkg/ui/styl/shame.styl, line 114 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-L0fM_w3WqG9oS6pWQih-r3-114:-L1seapwfrCbesjZajX0:b3zb47t) ([raw file](https://github.com/cockroachdb/cockroach/blob/553e24843d68b0f543cef1085d5919f497281788/pkg/ui/styl/shame.styl#L114)):*
<details><summary><i>Previously, couchand (Andrew Couch) wrote…</i></summary><blockquote>

is this change related?
</blockquote></details>

no, it's related to another change I made when the heading was not dark enough. Let me create a separate PR then.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825)*

**Comment by vilterp** [comment]
Probably an issue for another PR, but a useful thing to say here instead of just "count" would be the time bucketing, e.g. "count per 10s" or "queries per 10s".

I think it's hard to tell what bucketing is going on; adding it to the axis label could help.

**Comment by dianasaur323** [comment]
hmmm.. that would be potentially a quick fix, although I agree that it should be a separate PR. Probably related to that open issue about using bar charts instead of time series lines.

**Comment by dianasaur323** [comment]
@couchand any additional comments on this? I addressed most of your comments, but had some outstanding questions.

**Comment by couchand** [comment]
:lgtm: modulo the indentation of ComputeDurationScale.  thanks for taking care of this!

---

Reviewed 1 of 9 files at r1, 1 of 2 files at r3, 9 of 9 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---

*[pkg/ui/src/util/format.ts, line 80 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-L0f8Ikz3vqDoT_PgoiO-r3-80:-L2QzziD-E9Wu3KBcj_T:b-vmxc49) ([raw file](https://github.com/cockroachdb/cockroach/blob/553e24843d68b0f543cef1085d5919f497281788/pkg/ui/src/util/format.ts#L80)):*
<details><summary><i>Previously, dianasaur323 (Diana Hsieh) wrote…</i></summary><blockquote>

Could you clarify for my benefit? This looks similar to the below export function with a const scale declaration as well?
</blockquote></details>

I just mean that the indentation here is only one space instead of two.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-:-L2R-fJrVfeEWkCgaOX6:b-n02y6k)*

**Comment by dianasaur323** [comment]
Review status: 9 of 10 files reviewed at latest revision, 2 unresolved discussions.

---

*[pkg/ui/src/util/format.ts, line 80 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-L0f8Ikz3vqDoT_PgoiO-r3-80:-L2R3HHWbYN9rh5dAHdn:b8uq52w) ([raw file](https://github.com/cockroachdb/cockroach/blob/553e24843d68b0f543cef1085d5919f497281788/pkg/ui/src/util/format.ts#L80)):*
<details><summary><i>Previously, couchand (Andrew Couch) wrote…</i></summary><blockquote>

I just mean that the indentation here is only one space instead of two.
</blockquote></details>

ahhh i see! done!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825)*

**Comment by couchand** [comment]
:shipit: 

---

Reviewed 2 of 2 files at r5.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20825#-:-L2Vq1hAIlEK_9FZ2eYm:bizlbj6)*

**Pull request merged**
